libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -42,15 +41,6 @@ const AP_Param::GroupInfo AP_OAPathPlanner::var_info[] = {
     // @User: Standard
     AP_GROUPINFO_FLAGS("TYPE", 1,  AP_OAPathPlanner, _type, OA_PATHPLAN_DISABLED, AP_PARAM_FLAG_ENABLE),
 
-    // @Param: LOOKAHEAD
```

Comment:
We should add a comment so say that index "2" was used by LOOKAHEAD.  This warns future developers that they can't use "2" because if they did then users who upgraded their firmware could get parameter corruption.  It's not really severe corruption but a variable that used index 2 could get a funny value in it.